src/crimson/os/seastore/journal/record_submitter.cc

```diff
@@ -257,7 +257,7 @@ RecordSubmitter::roll_segment()
         wait_available_promise.reset();
       }).handle_error(
         crimson::ct_error::all_same_way([FNAME, this](auto e) {
-          ERROR("{} got error {}, available", get_name(), e);
+          //ERROR("{} got error {}, available", get_name(), e);
```

Comment:
Recall a previous commit https://github.com/ceph/ceph/pull/49420/commits/f99ebd9f14a6d3caebab9f1c5cbbca5b18c5a5d2, probably calling `e.what()` should be OK.

Reply:
Thank you. When trying to upgrade fmt, we had few instances of faulty log lines with probably missing formatters and so forth. I didn't try to fix each of them yet. This PR is an attempt to fix https://tracker.ceph.com/issues/66887. I'll un-comment in case the tests will pass.

Reply:
```
/home/matan/ceph/src/crimson/os/seastore/journal/record_submitter.cc:242:19:   required from here                                                                                                                                      
/home/matan/ceph/src/fmt/include/fmt/base.h:1641:63: error: ‘fmt::v11::detail::type_is_unformattable_for<std::error_code, char> _’ has incomplete type                                                                                 
 1641 |     type_is_unformattable_for<T, typename Context::char_type> _;                                                                                                                                                               
      |                                                               ^                                                                                                                                                                
/home/matan/ceph/src/fmt/include/fmt/base.h:1644:7: error: static assertion failed: Cannot format an argument. To make type T formattable provide a formatter<T> specialization: https://fmt.dev/latest/api.html#udt                   
 1644 |       formattable,                                                                                                                                                                                                             
      |       ^~~~~~~~~~~                                                                                                                                                                                                              
/home/matan/ceph/src/fmt/include/fmt/base.h:1644:7: note: ‘formattable’ evaluates to false
```